frontend/packages/console-app/src/components/modals/restore-pvc/restore-pvc-modal.tsx

```diff
@@ -144,9 +137,9 @@ const RestorePVCModal = withHandlePromise<RestorePVCModalProps>(
       );
     };
     return (
-      <div className="modal-content modal-content--no-inner-scroll">
-        <ModalTitle>{t('console-app~Restore as new PVC')}</ModalTitle>
-        <Form onSubmit={submit} name="form">
```

Comment:
moved Form to fix some glitch in UI

Reply:
> moved Form to fix some glitch in UI

Can you explain what the glitch is you needed to fix? It's unclear to me why moving the `Form` should have been necessary.

Reply:
Our existing pattern is to wrap `<ModalTitle>`, `<ModalBody>` and `<ModalFooter>` with `<form>`.  Note we have some bugs when trying to use `<Form className="modal-content modal-content--no-inner-scroll"`, so it's best to stick with `<form className="modal-content modal-content--no-inner-scroll"`